src/qml/QFieldCloudLogin.qml

```diff
@@ -73,6 +74,18 @@ Item {
         }
       }
 
+      Text {
+        text: qsTr( "This mobile device does not support QFieldCloud. Please upgrade your system to a newer Android version or use another device." )
```

Comment:
```suggestion
        text: qsTr( "This mobile device runs an Android version that is unstable with SSL connections and therefore cannot connect to QFieldCloud. Please upgrade your system to a newer Android version or use another device." )
```

@signedav what do you think about the wording?

Reply:
As far as I understand it it's not the Android version. Should we say an Android *installation*? Because otherwise the clients are confused since it runs on one Android 8 Version but not on the other.

Reply:
Device?